collector/analyzer/pgftmetricanalyzer/pgft_analyzer.go

```diff
@@ -0,0 +1,137 @@
+package pgftmetricanalyzer
+
+import (
+	"github.com/Kindling-project/kindling/collector/analyzer"
+	"github.com/Kindling-project/kindling/collector/component"
+	"github.com/Kindling-project/kindling/collector/consumer"
+	conntrackerpackge "github.com/Kindling-project/kindling/collector/metadata/conntracker"
+	"github.com/Kindling-project/kindling/collector/model"
+	"github.com/Kindling-project/kindling/collector/model/constlabels"
+	"github.com/Kindling-project/kindling/collector/model/constnames"
+	"github.com/hashicorp/go-multierror"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	PgftMetric analyzer.Type = "pgftmetricanalyzer"
+)
+
+var consumableEvents = map[string]bool{
+	constnames.SwitchEvent: true,
+}
+
+type PgftMetricAnalyzer struct {
+	consumers   []consumer.Consumer
+	conntracker conntrackerpackge.Conntracker
```

Comment:
You won't need `conntracker` since your metric is none of business with the network.